vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -74,6 +74,12 @@ NS_SWIFT_NAME(BraveAds)
 /// Whether or not Brave Ads is enabled
 @property (nonatomic, assign, getter=isEnabled) BOOL enabled;
 
+/// Whether or not the user has opted out of subdevision ad targeting
+@property (nonatomic, assign, getter=shouldAllowSubdivisionAdTargeting) BOOL allowSubdivisionAdTargeting;
```

Comment:
nit: `allowSubdivisionAdTargeting` could also be `shouldAllowSubdivisionAdTargeting`